### ISSUE TITLE
Changes to webpage

### ISSUE THREAD

**Issue by deekej** [kejbaly2/devconfcz#17]
Hey,

I'm dumping here some ideas on how to improve the webpage, since I do not know much about CSS... :-/ If anyone finds anything else, feel free to add comment here as well. :)

From mailing list:
----------------------
One more thing - there's a lot of text, which seems to be unstructurized (continous flow of it):
* Session Types, Difficulty Levels, ... - distinguish them as heading
* maybe use italics for *Keynotes*, *Talks*, *Workshop / Lab* so they are more easily dinstinguish by the first look?
* put the links to youtube videos below each other (the names are quite long, it's hard to follow them if they are behind each other)

**Comment by kejbaly2** [comment]
let me know if these are still valid and if so, which. Thanks.  Check latest version at 

https://kejbaly2.github.io/devconfcz/index.html

**Comment by kejbaly2** [comment]
Re-open or create a new issue if any of these are still valid.
